doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -151,6 +151,7 @@ Bluetooth samples
   * :ref:`peripheral_hids_mouse` - Added a notice about encryption requirement.
   * :ref:`peripheral_hids_keyboard` - Added a notice about encryption requirement.
   * :ref:`central_and_peripheral_hrs` - Clarified the "Requirements" section and fixed a link in the "Testing" section of the document.
+  * :ref:`peripheral_uart` - Increased system workqueue stack size for prj_minimal.conf.
```

Comment:
```suggestion
  * :ref:`peripheral_uart` - Increased the system workqueue stack size for the :file:`prj_minimal.conf` file.
```
Should we be more precise and say what the new stack size is?

Reply:
I don't think it's necessary to write precise number.